packages/now-cli/src/util/input/input-project.ts

```diff
@@ -82,7 +82,7 @@ export default async function inputProject(
     const answers = await inquirer.prompt({
       type: 'input',
       name: 'newProjectName',
-      message: `How would you like to call your new project?`,
+      message: `What’s your project’s name?`,
     });
```

Comment:
I wonder if we should default to the directory name or what we find in `now.json` if one exists

Reply:
Agreed, it should definitely use `now.json`'s `name` property if it exists. There's no reason to ask.

If there is no `name`, the default should be the current directory name.

Reply:
Let's merge that first (it's simply changing the label of the question) and continue this discussion. I think we can do that (have a default name), but there's some case where it's not possible and we need a fallback. I'll open a discussion on Slack 👍